### ISSUE TITLE
[A11Y] - No water wise

### ISSUE THREAD

**Issue by DeclanMorrison** [The-First-Pancake/cappy_and_tappy#23]
**Describe the feature and who it benefits**
Ability to turn off water rising, helps casual players and those who are unable to play at a rate fast enough to outrun the water
